modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -218,51 +470,67 @@ cv::Vec3i HashTSDFVolumeCPU::volumeToVoxelCoord(cv::Point3f point) const
                      cvFloor(point.z * voxelSizeInv));
 }
 
+inline TsdfVoxel _at(const cv::Vec3i& volumeIdx, InputArray _volume,
```

Comment:
As you mentioned, this part can be optimized to avoid that `_volume` argument passing (by passing index in volumes for example)

Reply:
done